### ISSUE TITLE
Hello I'm using this npm package and i'm wondering if there is a way to turn off the "source" tags, you know how there are `source` below each jsdoc definition how do i turn that off? i have read through the docs and i don't know if i'm blind or something but i just can't find any way to do that

### ISSUE THREAD

**Issue by TakenKills** [ankitskvmdam/clean-jsdoc-theme#60]

**Comment by ankitskvmdam** [comment]
Thanks for creating this issue. I saw that currently, we are enforcing to show source code. But we will try to remove this rule in the future release which you can expect soon. After the release, you can use jsdoc config to hide the source code.

**Comment by TakenKills** [comment]
Ahh got it thanks for the info

**Comment by ankitskvmdam** [comment]
New `clean-jsdoc-theme` version is released, update to `3.0.10`, and then you are able to turn off/hide the source.
You have to update your jsdoc configuration:
```js
templates: {
     defaults: {
          outputSourceFiles: false
     }
}
```

for more information visit: [configure default template](https://jsdoc.app/about-configuring-default-template.html), [jsdoc-config](https://jsdoc.app/about-configuring-jsdoc.html)

**Comment by TakenKills** [comment]
Oh, Thanks!
